cmd/crio/main.go

```diff
@@ -63,22 +63,22 @@ func catchShutdown(ctx context.Context, cancel context.CancelFunc, gserver *grpc
 			case unix.SIGPIPE:
 				continue
 			case signals.Interrupt:
-				logrus.Debugf("Caught SIGINT")
+				log.Debugf(ctx, "Caught SIGINT")
```

Comment:
@saschagrunert are the logging interceptors registered by this point? (I don't know how they work)

Reply:
It looks like yes, the interceptors are registered by this point (and it seems like done a few lines before catchShutdown() is registered), but please, wait for @saschagrunert's input (as I'm probably wrong here ;-))

Reply:
haha I believe you, I was being lazy and didn't check, so the fact you checked means it's there :+1: thanks :D